packages/client/src/views/Performance/RegistrationsList.tsx

```diff
@@ -395,139 +399,145 @@ function RegistrationListComponent(props: IProps) {
+    result: RegistrationsListByRegistrarFilter['results'][0],
+    index: number
+  ) => ({
+    ...result,
+    name: (
+      <Stack>
+        <AvatarSmall
+          name={
+            result.registrarPractitioner?.name
+              ? getName(result.registrarPractitioner.name, 'en')
+              : ''
+          }
+          avatar={result.registrarPractitioner?.avatar}
+        />
+        <>
+          {!isPerformanceManager ? (
+            <Link
+              font="bold14"
+              onClick={() => {
+                props.goToUserProfile(String(result.registrarPractitioner?.id))
+              }}
+            >
+              {result.registrarPractitioner?.name
+                ? getName(result.registrarPractitioner.name, 'en')
+                : ''}
+            </Link>
+          ) : (
+            String(
+              result.registrarPractitioner?.name
+                ? getName(result.registrarPractitioner.name, 'en')
+                : ''
+            )
+          )}
+        </>
+      </Stack>
+    ),
+    location: (
+      <>
+        {!isPerformanceManager ? (
+          <Link
+            font="bold14"
+            onClick={() => {
+              props.goToTeamUserList(
+                result.registrarPractitioner?.primaryOffice?.id as string
+              )
+            }}
+          >
+            {result.registrarPractitioner?.primaryOffice?.name}
+          </Link>
+        ) : (
+          String(result.registrarPractitioner?.primaryOffice?.name)
+        )}
+      </>
+    ),
+    systemRole: getFieldAgentTypeLabel(
+      result.registrarPractitioner?.systemRole as string
+    ),
+    total: String(result.total),
+    delayed: showWithTooltip(result.total, result.delayed, 'delayed', index),
+    delayed_num: getPercentage(result.total, result.delayed),
+    late: showWithTooltip(result.total, result.late, 'late', index),
+    late_num: getPercentage(result.total, result.late)
+  })
+
+  const getTableContentByLocation = (
+    result: RegistrationsListByLocationFilter['results'][0],
+    index: number
+  ) => ({
+    ...result,
+    location: result.location.name,
+    total: String(result.total),
+    delayed: showWithTooltip(result.total, result.delayed, 'delayed', index),
+    delayed_num: getPercentage(result.total, result.delayed),
+    late: showWithTooltip(result.total, result.late, 'late', index),
+    late_num: getPercentage(result.total, result.late),
+    home: showWithTooltip(result.total, result.home, 'home', index),
+    home_num: getPercentage(result.total, result.home),
+    healthFacility: showWithTooltip(
+      result.total,
+      result.healthFacility,
+      'healthFacility',
+      index
+    ),
+    healthFacility_num: getPercentage(result.total, result.healthFacility)
+  })
+
+  const getTableContentByTime = (
+    result: RegistrationsListByTimeFilter['results'][0],
+    index: number
+  ) => {
+    return {
+      ...result,
+      // Time is epoch but returned as a string
```

Comment:
This line should be the only changed one. Otherwise I just created a wrapper function for each case and adjusted to proper typing.